Makefile

```diff
@@ -76,16 +76,28 @@ ${BIN_DIR}/deepcopy-gen:
 	$(info Compiling deepcopy-gen...)
 	go build -o ${BIN_DIR}/deepcopy-gen ./cmd/deepcopy-gen/
 
+
+# Build the docker image and tag it.  
 images: verify-tag-name generate-code update-deployment-crds
 	$(info List executable directory)
 	$(info repo id: ${git_repository_id})
 	$(info branch: ${GIT_BRANCH})
 	$(info Build the docker image)
-ifeq ($(strip $(GO_BUILD_ARGS)),)
-	docker build --quiet --no-cache --tag mcad-controller:${TAG} -f ${CURRENT_DIR}/Dockerfile  ${CURRENT_DIR}
-else 
-	docker build --no-cache --tag mcad-controller:${TAG} --build-arg GO_BUILD_ARGS=$(GO_BUILD_ARGS) -f ${CURRENT_DIR}/Dockerfile  ${CURRENT_DIR}
-endif		
+	ARCH := $(shell uname -m)
+	ifeq ($(ARCH),aarch64)
+		ifeq ($(strip $(GO_BUILD_ARGS)),)
+			docker buildx  build --quiet --no-cache --platform=linux/amd64 --tag mcad-controller:${TAG} -f ${CURRENT_DIR}/Dockerfile  ${CURRENT_DIR}
+		else 
+			docker buildx build --no-cache --platform=linux/amd64 --tag mcad-controller:${TAG} --build-arg GO_BUILD_ARGS=$(GO_BUILD_ARGS) -f ${CURRENT_DIR}/Dockerfile  ${CURRENT_DIR}
+		endif	
+	else
+		ifeq ($(strip $(GO_BUILD_ARGS)),)
+			docker build --quiet --no-cache --tag mcad-controller:${TAG} -f ${CURRENT_DIR}/Dockerfile  ${CURRENT_DIR}
+		else 
+			docker build --no-cache --tag mcad-controller:${TAG} --build-arg GO_BUILD_ARGS=$(GO_BUILD_ARGS) -f ${CURRENT_DIR}/Dockerfile  ${CURRENT_DIR}
+		endif	
+	endif
```

Comment:
Tested locally on my x86 machine. 

I'm getting a syntax error. It seems the indentation here is making the Makefile think the `ifeq` and other conditional statements to be treated as shell commands. The solution:

Unindent ALL conditional statements to the very beginning of the line.

Reply:
This might be a better solution for readability: instead of unindenting everything, indent the conditional statements and their content with Spacebar spaces instead of using the `TAB` key.

Makefiles can be strange like this sometimes :)